content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -144,7 +144,7 @@ tolerations:
   tolerationSeconds: 3600
 ```
 
-means that if this pod is running and a matching taint is added to the node, then
+means that if this pod is running and a non-tolerated taint is added to the node, then
```

Comment:
The YAML above is only about the "matching" taint. The item is under "tolerations", so its semantics is pretty clear. In this context, the text was trying to explain how `tolerationSeconds` works. It has nothing to do about a different taint that is not tolerated. So this "fix" is invalid.

Hope this clarifies the reason I closed this PR.

Reply:
Thanks for the reasoning.